libcrux-intrinsics/Cargo.toml

```
@@ -9,12 +9,6 @@ repository.workspace = true
 readme.workspace = true
 
```

```suggestion
exclude = [ "/proofs" ]

```
We want to exclude proofs from the published crates, also on the other ones.